examples/graphql-shield-authorization.js

```diff
@@ -1,126 +0,0 @@
-'use strict'
```

Comment:
it's dependencies are not yet compatible with graphql 16

Discussion here --> https://github.com/mercurius-js/mercurius/issues/628#issuecomment-967035794

Reply:
Cool thanks!

Reply:
rather than delete, isn't this something we can upgrade to work with graphql@16? In any case we should think about a better way other than deleting. Even if they are not compatible, I assume they will be at some point, meaning that we may want to keep the example around in some way